test/unit/clarity_ext/dilution/test_dilutions.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+from mock import MagicMock
+from clarity_ext.dilution import DilutionScheme
+from test.unit.clarity_ext.helpers import fake_analyte
+
+
+class TestDilutionScheme(unittest.TestCase):
+    """
+    Validates that the DilutionScheme property of the context object returns the expected values
+    based on fake data from the LIMS
+    """
+
+    def test_dilution_scheme_hamilton_expected_results(self):
+        """Dilution scheme created by mocked analytes is correctly generated for Hamilton"""
+        # Setup:
+        repo = MagicMock()
+        repo.all_analytes = all_analyte_test_set1
+        dilution_scheme = DilutionScheme(repo, "Hamilton")
+
+        expected = [
+            ['art-name1', '36', 'DNA1', '14.9', '5.1', '34', 'END1'],
+            ['art-name2', '33', 'DNA2', '14.9', '5.1', '17', 'END2'],
+            ['art-name3', '50', 'DNA2', '14.9', '5.1', '44', 'END1'],
+            ['art-name4', '93', 'DNA2', '14.9', '5.1', '69', 'END2']]
+
+        # Test:
+        actual = [
+            [dilute.sample_name,
+             "{}".format(dilute.source_well_index),
+             dilute.source_plate_pos,
+             "{:.1f}".format(dilute.sample_volume),
+             "{:.1f}".format(dilute.buffer_volume),
+             "{}".format(dilute.target_well_index),
+             dilute.target_plate_pos] for dilute in dilution_scheme.dilutes
+        ]
```

Comment:
... so I think of this kind of duplication as "forking" rather than just blind duplication, since each part will live their separate lives from then on.